### DISCUSSION TITLE
Is ALIAS DNS record type supported?

### DISCUSSION THREAD

**Discussion by kemeris2000** [sys4/netbox-plugin-dns#31] [Q&A] [answered]
Maybe it is under different name?

**Comment by peteeckel** [comment, marked as answer]
NetBox DNS supports all RR types that `dnspython` supports as well, which is pretty much what the RFCs have to offer. For all I know there is no `ALIAS` RR type - do you have a reference for that?

That said, I guess you're referring to `CNAME`, which is supported.

**Reply by kemeris2000** [reply]
Just checked RFC 1034. You are right, ALIAS is pseudo record and is not part of the DNS specification.
